pkg/storj/serialnumber.go

```diff
@@ -44,6 +44,18 @@ func (id SerialNumber) IsZero() bool {
 	return id == SerialNumber{}
 }
 
+// Less returns whether id is smaller than b in lexiographic order
+func (id SerialNumber) Less(b SerialNumber) bool {
```

Comment:
nit - why is b the name for the SerialNumber argument?

Reply:
Because I couldn't figure out a better name.

Reply:
not the best name, but `another` or `other` looks better to me.

Reply:
used `other`